docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+services:
+  nats:
+    image: nats:latest
```

Comment:
I wonder if we should peg this to a version like we do in the CircleCI config? I've been keeping the version documented in the README, but that way we are disclosing what we test against.

It might be more valuable though to just attempt to always support the `latest` 🤔